test/blueprints/Do.Test.Blueprints.Service/Authentication/Authentication.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Do.Test;
```

Comment:
you need to adjust namespaces to fit their folder, `Do.Test.Authentication` in this case